### PR TITLE
Additional check for the localizeCache

### PR DESCRIPTION
**Pull request by monapochi**
This is a fix proposal for [PolymerElements/app-localize-behavior#65](PolymerElements/app-localize-behavior#65)

P.S. I'm sorry for my previous mis-PR

### CODE REVIEW

File: app-localize-behavior.html [hunk 1]
```diff
@@ -254,6 +255,16 @@
 
   __onRequestError: function(event) {
     this.fire('app-localize-resources-error');
+  },
+
+  __checkLocalizationCache(proto) {
+    // do nothing if proto is undefined.
+    if(proto === undefied) return;
```
Comment (jimivdw) [inline app-localize-behavior.html]: You have a typo here. `undefied` should, of course, be `undefined`

Comment (amohabirQAFE) [inline app-localize-behavior.html]: Typo in here: undefied --> undefined

**Comment by monapochi** [comment]
Silly me!
Thanks  a lot, @jimivdw, @amohabirQAFE, I'll fix it soon.

**Comment by notwaldorf** [comment]
Thanks!

**Pull request merged**